### ISSUE TITLE
MAS 1.4.3: Elements must have sufficient color contrast (.active)

### ISSUE THREAD

**Issue by Amulya-95** [terrajobst/apisof.net#22]
**GitHubTags:**

#A11yTCS; #A11yMAS; #A11yAuto; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS1.4.3; #C+E-H; #api-catalog; #A11ySev2; #Benchmark;

#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[System.Activities.Debugger - .NET API Catalog](https://apisof.net/catalog/System.Activities.Debugger)
#### Element path
.active
#### Snippet
`<li class="active"><img width="16" height="16" src="/images/api/Namespace_16x.svg"> System.Activities.Debugger</li>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.1 (foreground color: #777777, background color: #f5f5f5, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 76.0.3809.100
This accessibility issue was found using Accessibility Insights for Web 2.8.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![Keros bug (MAS1 4 3)](https://user-images.githubusercontent.com/51435266/63583760-3e847300-c5b9-11e9-8015-05f697742d2e.png)

**Comment by terrajobst** [comment]
The web site no longer exists.
